README.md

```diff
@@ -93,28 +93,27 @@ while a subscriber efficiently receives and prints the data.
 use elkodon::prelude::*;
 use elkodon_bb_posix::signal::SignalHandler;
 
-fn main() {
-    let service_name = ServiceName::new(b"My/Funk/ServiceName").unwrap();
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let service_name = ServiceName::new(b"My/Funk/ServiceName")?;
 
     let service = zero_copy::Service::new(&service_name)
         .publish_subscribe()
-        .open_or_create::<usize>()
-        .expect("failed to create/open service");
+        .open_or_create::<usize>()?;
 
-    let publisher = service
-        .publisher()
-        .create()
-        .expect("failed to create publisher");
+    let publisher = service.publisher().create()?;
 
     while !SignalHandler::termination_requested() {
-        let mut sample = publisher.loan().expect("Failed to acquire sample");
-        unsafe { sample.write(1234); }
-        publisher.send(sample).expect("Failed to send sample");
+        let mut sample = publisher.loan()?;
+        unsafe {
+            sample.as_mut_ptr().write(1234);
+        }
+        publisher.send(sample)?;
```

Comment:
This API is currently unsound since it introduces undefined behavior without `unsafe`, e.g.
```rust
let sample = publisher.loan();
publisher.send(sample);
```

Shall we implement a similar API like the one in iceoryx-rs before the official announcement?

Reply:
Yes, I added this as a todo note in #3